### ISSUE TITLE
saturn moons

### ISSUE THREAD

**Issue by johndpope** [ThumbWorks/AugmentedSolarSystem#45]
https://en.wikipedia.org/wiki/Moons_of_Saturn
![pjgnon9](https://user-images.githubusercontent.com/289994/32743805-86005a56-c87b-11e7-8bd2-0a4791cd194c.gif)

**Comment by johndpope** [comment]
fyi - http://physics.unl.edu/~astrodev/flashdev2/ 
there's a treasure trove of pretty flash animations here maybe able to port some of these over

**Comment by rodericj** [comment]
@johndpope yeah great suggestion. I think this could be handled with #41. The logic should be roughly the same and SwiftAA handles them I believe. 

tl;dr, I'm down.
